src/main/java/wolox/training/models/User.java

```diff
@@ -0,0 +1,99 @@
+package wolox.training.models;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+import wolox.training.exceptions.BookAlreadyOwnedException;
+
+@Entity
+@Table(name="Users")
+public class User{
+	
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+    
+    @NotNull
+    private String userName;
+    
+    @NotNull
+    private String name;
+    
+    @NotNull
+    private Date birthDate;
+    
+    @ManyToMany(cascade = {CascadeType.REFRESH, CascadeType.MERGE})
+//    @JoinTable(name = "user_book", 
+//      joinColumns = @JoinColumn(name = "user_id", referencedColumnName = "id"), 
+//      inverseJoinColumns = @JoinColumn(name = "book_id", 
+//      referencedColumnName = "id"))
+    private List<Book> books = new ArrayList<Book>();
+
+    
+    public User() {
+    	
+    }
+
+    public long getId() {
+    	return id;
+    }
+
+	public String getUsername() {
+		return userName;
+	}
+
+	public void setUsername(String username) {
+		this.userName = username;
+	}
+
+	public String getName() {
+		return name;
+	}
+
+	public void setName(String name) {
+		this.name = name;
+	}
+
+	public Date getBirthDate() {
+		return birthDate;
+	}
+
+	public void setBirthDate(Date birthDate) {
+		this.birthDate = birthDate;
+	}
+
+	public List<Book> getBooks() {
+		return (List<Book>) Collections.unmodifiableList(books);
+	}
+
+	public void setBooks(List<Book> books) {
+		this.books = books;
+	}
+    
+    public List<Book> AgregarLibro(Book userBook) throws BookAlreadyOwnedException{

```

Comment:
Put the name of the method in English.

Reply:
Es necesario?

Reply:
Si por convención lo nombres de los métodos van en ingles y con su inicial en minúscula. Si te sirve podrías dejar algún comentario sobre el método en español de que es lo que hace el método.

Reply:
Done